pkg/qemu/qemu.go

```diff
@@ -441,14 +458,28 @@ func adjustMemBytesDarwinARM64HVF(memBytes int64, accel string, features *featur
 	return memBytes
 }
 
+func getMachine(arch limayaml.Arch) string {
+	switch arch {
+	case limayaml.X8664:
+		return "q35"
+	case limayaml.AARCH64:
+		return "virt"
+	case limayaml.ARMV7L:
+		return "virt"
+	case limayaml.RISCV64:
+		return "virt"
+	}
```

Comment:
```go
switch arch {
  case limayaml.X8664:
    return "q35"
  case limayaml.AARCH64, limayaml.ARMV7L, limayaml.RISCV64:
    return "virt"
  default: // what?
    return "none"
}
```

Reply:
I'm not sure why there was a compiler warning when testing all possibilities of an enum?

But maybe it was because using a magic string instead, either way it "shouldn't happen"